### PR TITLE
Disambiguate untitled applications

### PR DESCRIPTION
**Pull request by davidrapson**
Disambiguate untitled applications using a suffix

image

### CODE REVIEW

File: views/components/application-card/macro.njk [hunk 1]
```diff
@@ -58,7 +69,9 @@
                 <a href="{{ localify(props.editUrl) }}"
                    class="btn btn--small">
                     {{ cardCopy.continue }}
-                    <span class="u-visually-hidden">({{ projectName | safe }})</span>
+                    <span class="u-visually-hidden">
+                        ({{ props.projectName or props.untitledName }})
```
Comment (mattandrews) [inline views/components/application-card/macro.njk]: I think the initial a11y report was pointing out that a list of untitled applications will always read out "Untitled proposal" at this point, so could we add the datestring into this label?

Comment (davidrapson) [reply views/components/application-card/macro.njk]: Good shout, I did wonder that as I was updating this.

**Comment by davidrapson** [comment]
[Add a unique identifier for untitled applications on the dashboard](https://trello.com/c/Lq2afSAj/969-add-a-unique-identifier-for-untitled-applications-on-the-dashboard)

**Comment by davidrapson** [comment]
Added date to assistive label and just waiting on a small bit of welsh copy before I merge this.

**Pull request merged**